docs/source/jobs_training.md

```diff
@@ -0,0 +1,352 @@
+# Training using Jobs
+
+[Jobs](https://huggingface.co/docs/huggingface_hub/guides/jobs) lets you run training scripts on fully managed infrastructure (no need to handle GPUs, dependencies, or environment setup locally). This makes it easy to scale and monitor your experiments directly from the Hub.
+
+In this guide, you’ll learn how to:
+- Run TRL training scripts using Jobs.
+- Configure hardware, timeouts, environment variables, and secrets.
+- Monitor and manage jobs from the CLI or Python.
+
+**Requirements**
+- Pro, Team, or Enterprise plan.
+- Logged into the Hugging Face Hub (`hf auth login`).
+
+## Preparing your Script
+
+You can launch Jobs using either the [`hf jobs` CLI](https://huggingface.co/docs/huggingface_hub/guides/cli#hf-jobs) or the Python API. A convenient option is to use [UV scripts](https://docs.astral.sh/uv/guides/scripts/), which packages all dependencies directly into a single Python file. You can run them like this:
+
+<hfoptions id="script_type">
+<hfoption id="bash">
+
+```bash
+hf jobs uv run --flavor a10g-small "https://raw.githubusercontent.com/huggingface/trl/main/trl/scripts/sft.py" 
```

Comment:
as a newby, I'll try to run this, but it will most likely fail, because you're not providing a dataset or a model. Can you add a  `--model_name_or_path`  and `dataset_name` that would work?

Reply:
I think the above comment doesn't  apply to `hf jobs uv run --flavor a10g-small sft.py` though